test/e2e/ansible/suite_test.go

```diff
@@ -60,6 +60,11 @@ var _ = BeforeSuite(func() {
 	By("copying sample to a temporary e2e directory")
 	Expect(exec.Command("cp", "-r", "../../../testdata/ansible/memcached-operator", tc.Dir).Run()).To(Succeed())
 
+	By("enabling debug logging in the manager")
+	err = testutils.ReplaceInFile(filepath.Join(tc.Dir, "config", "default", "manager_auth_proxy_patch.yaml"),
+		"- \"--enable-leader-election\"", "- \"--enable-leader-election\"\n        - \"--zap-log-level=2\"")
+	Expect(err).NotTo(HaveOccurred())
+
 	By("fetching the current-context")
 	tc.Kubectx, err = tc.Kubectl.Command("config", "current-context")
 	Expect(err).NotTo(HaveOccurred())
```

Comment:
It is missing the fragment changelog. Could you please add as well?
See; https://github.com/operator-framework/operator-sdk/blob/master/changelog/fragments/00-template.yaml

it would like : 

```
entries:
  - description: >
      For Ansible-based operators, change the level of the proxy skipping cache logs to debug level.  
    kind: "change"
    breaking: false
```